src/main/java/org/generationcp/middleware/liquibase/UpdatePresetsTask.java

```diff
@@ -0,0 +1,157 @@
+package org.generationcp.middleware.liquibase;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import liquibase.database.Database;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import org.generationcp.middleware.domain.labelprinting.LabelPrintingPresetDTO;
+import org.generationcp.middleware.domain.labelprinting.PresetDTO;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UpdatePresetsTask implements liquibase.change.custom.CustomTaskChange {
```

Comment:
@cuenyad it will be interesting to add tests for this task

Reply:
I will do it.

Reply:
maybe we can group this differently given that it's seems to be a one time change, right?, maybe another package? like `src/main/java/org/generationcp/middleware/liquibase/tasks/v23/UpdatePresetsTask.java`
if it were a reusable tasks I guess a version would not be necessary, though probably it would be risky to reuse this tasks because of complexity and backward breaking changing issues.
I guess a custom stored procedure was not possible, right?

cc @cuenyad @clarysabel

Reply:
@nahuel-soldevilla agree on the suggested package, @cuenyad please move the class 
and right, the only reason to create this task is the json parsing

Reply:
I will do it.